src/modules/mc_att_control/mc_att_control_params.c

```diff
@@ -538,6 +538,8 @@ PARAM_DEFINE_FLOAT(MC_TPA_RATE_D, 0.0f);
  * The D-term uses the derivative of the rate and thus is the most susceptible to noise.
  * Therefore, using a D-term filter allows to decrease the driver-level filtering, which
  * leads to reduced control latency and permits to increase the P gains.
+ * A good rule of thumb is to fix the time constant of this filter equal to one tenth of the
+ * derivative time constant: MC_DTERM_CUTOFF = 10 * MC_(ROLL|PITCH)RATE_P / (2 * pi * MC_(ROLL|PITCH)RATE_D)
```

Comment:
This formula gives too high values for the setups I looked at (about 130). Also, setting this properly depends on the vibration frequencies, which are not in the formula, so I don't think it's very helpful advice.

Reply:
This rule means "If you choose that cutoff, you are sure that the filter will not add too much phase lag". I checked and in the config files we have more
`MC_DTERM_CUTOFF ~= 5 * MC_(ROLL|PITCH)RATE_P / (2 * pi * MC_(ROLL|PITCH)RATE_D)`
Maybe having that factor 5 is better for our setups given the amount of noise.
Should we remove completely the rule of thumb of the parameter description and add it to the documentation when we have more tests on that equation?

Reply:
If it means something different, why didn't you write that then? :)
Yes it sounds like it should be in the documentation, and more in detail as to what it means. And since we're lower by a factor of 2, and it still works fine, the formula doesn't seem to be a strict bound, so I question a bit its usefulness.